src/extension/popup-page/index.tsx

```diff
@@ -65,7 +65,16 @@ function PopupUI() {
 
     const ui = getActivatedUI()
     const myIdentities = useValueRef(ui.myIdentitiesRef)
-    const openOptionsPage = () => browser.runtime.openOptionsPage()
+    const openOptionsPage = (event: React.MouseEvent) => {
+        if (event.shiftKey) {
+            browser.tabs.create({
```

Comment:
Why not use normal `open`?

Reply:
open = new window
the method is create a new tab